src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -384,7 +316,7 @@ public final Response createGameboard(
       return new SegueErrorResponse(Status.BAD_REQUEST, "No wildcard available. Unable to construct gameboard.")
           .toResponse();
     } catch (InvalidGameboardException e) {
-      return new SegueErrorResponse(Status.BAD_REQUEST, String.format("The gameboard you provided is invalid"), e)
```

Comment:
Correct me if I'm wrong, but it looks like this `InvalidGameboardException` is used to display an informative message to the user regarding what was wrong with the provided game board. So by removing the exception from the response, the user won't see that message. I think it's fine to remove the `String.format` bit, as the string is not really being formatted, but the exception should still be passed to the `SegueErrorResponse`.